tests/accessor/accessor_iterator_requirement.cpp

```diff
@@ -0,0 +1,155 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification that accessor's iterator is conforming to named
+//  requirement LegacyRandomAccessIterator
+//
+*******************************************************************************/
+#include "../../util/named_requirement_verification/legacy_random_access_iterator.h"
+#include "../common/disabled_for_test_case.h"
+#include "catch2/catch_test_macros.hpp"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+#include "accessor_common.h"
+#endif
+
+namespace accessor_iterator_requirement {
+
+using string_view = std::basic_string_view<char>;
+
+/**
+ * @brief Function helps to fail catch2 test and print errors from array through
+ * a INFO invocations
```

Comment:
```suggestion
 * @brief Function helps to fail catch2 test and print errors from array through
 *             a INFO invocations
```

Reply:
Why do we need all these indents? They doesn't appear if I'm using clang format.

Reply:
I don't know, we use formatting with indents in all other files